polkadot-parachain/src/service.rs

```diff
@@ -1165,8 +1165,9 @@ where
 	))
 }
 
-/// Start a statemint/statemine/westmint parachain node.
-pub async fn start_statemint_node<RuntimeApi, AuraId: AppKey>(
+/// Start an aura powered parachain node.
+/// (collective-polkadot and statemine/t use this)
```

Comment:
wonder if it worth on the api side to mention the clients.
hard to maintain and gets outdated.
meanwhile easy to check the usage/clients within one repo.

Reply:
I want to make it clear that if you mess with this function you're going to be impacting parachains X, Y and Z. If we get to the point where we have lots of parachains using this and everyone knows to be uber-careful with this function then we can delete the comment. But for now I'd like to have it there so people don't have to figure this out for themselves.